questionnaire/form-helper.js

```diff
@@ -165,7 +167,13 @@ function correctPartialDates(body, questionId) {
 }
 
 function addPrefix(section) {
-    return sectionList[section];
+    if (section === 'system') {
+        return section;
+    }
+    if (section.substring(0, 4) === 'info') {
```

Comment:
As above, [startsWith](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) for better readability.

For things like this, always be explicit. The function should communicate that it replaces "info-" with "p--". Although it does do that, it's not so readable, as it's intentionally leaving a "-" so that the replacement "p-" becomes "p--". Another issue with omitting the delimiter, is that it could unintentionally pickup other IDs e.g.

"p-informal-description-of-crime" (totally contrived ID)

Reply:
Changed replace to startsWith and changed to include -